designer/server/plugins/routes/api.ts

```diff
@@ -68,6 +73,10 @@ export const putFormWithId: ServerRoute = {
         await publish(id, value);
         return h.response({ ok: true }).code(204);
       } catch (err) {
+        request.logger.error(
+          ["newConfig", "previewModeError"],
+          consoleMessages.publishError
+        );
```

Comment:
copypasta? This is just getting a form with id. not always to do with newConfig? There's also no confirmation if this is a "previewModeError". you would have to do error.statusCode === 404 (since this is the statusCode boom.notFound returns).